[2022-07-22 12:59:39,476] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-09T00:00:00+00:00 [queued]>
[2022-07-22 12:59:41,766] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-09T00:00:00+00:00 [queued]>
[2022-07-22 12:59:41,790] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 12:59:41,800] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-22 12:59:41,806] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 12:59:42,459] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t2> on 2022-06-09 00:00:00+00:00
[2022-07-22 12:59:42,535] {standard_task_runner.py:52} INFO - Started process 735 to run task
[2022-07-22 12:59:42,609] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 't2', 'scheduled__2022-06-09T00:00:00+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/nickdag.py', '--cfg-path', '/tmp/tmpyu6_nspn', '--error-file', '/tmp/tmpkhbc83c2']
[2022-07-22 12:59:42,667] {standard_task_runner.py:80} INFO - Job 17: Subtask t2
[2022-07-22 12:59:42,800] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-22 12:59:43,461] {task_command.py:371} INFO - Running <TaskInstance: tutorial.t2 scheduled__2022-06-09T00:00:00+00:00 [running]> on host 2f63307804d5
[2022-07-22 12:59:44,324] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caroline.kosalka@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=t2
AIRFLOW_CTX_EXECUTION_DATE=2022-06-09T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-09T00:00:00+00:00
[2022-07-22 12:59:44,328] {python.py:173} INFO - Done. Returned value was: goodbye
[2022-07-22 12:59:44,604] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=t2, execution_date=20220609T000000, start_date=20220722T125939, end_date=20220722T125944
[2022-07-22 12:59:45,418] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-22 12:59:46,274] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-07-25 09:17:56,051] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-09T00:00:00+00:00 [queued]>
[2022-07-25 09:17:56,101] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-09T00:00:00+00:00 [queued]>
[2022-07-25 09:17:56,111] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 09:17:56,112] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-25 09:17:56,114] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 09:17:56,183] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t2> on 2022-06-09 00:00:00+00:00
[2022-07-25 09:17:56,209] {standard_task_runner.py:52} INFO - Started process 706 to run task
[2022-07-25 09:17:56,242] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 't2', 'scheduled__2022-06-09T00:00:00+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/nickdag.py', '--cfg-path', '/tmp/tmpt1pvvs7r', '--error-file', '/tmp/tmpl09uctmh']
[2022-07-25 09:17:56,251] {standard_task_runner.py:80} INFO - Job 25: Subtask t2
[2022-07-25 09:17:56,333] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-25 09:17:56,773] {task_command.py:371} INFO - Running <TaskInstance: tutorial.t2 scheduled__2022-06-09T00:00:00+00:00 [running]> on host 01e6b1df1873
[2022-07-25 09:17:57,619] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caroline.kosalka@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=t2
AIRFLOW_CTX_EXECUTION_DATE=2022-06-09T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-09T00:00:00+00:00
[2022-07-25 09:17:57,622] {python.py:173} INFO - Done. Returned value was: goodbye
[2022-07-25 09:17:57,810] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=t2, execution_date=20220609T000000, start_date=20220725T091756, end_date=20220725T091757
[2022-07-25 09:17:57,899] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-25 09:17:58,020] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
